### DISCUSSION TITLE
Do I have consider sequence/acknowledgement number overflow?

### DISCUSSION THREAD

**Discussion by kmc7468** [ANLAB-KAIST/KENSv3#209] [Project Q&A 2. Transport Layer (TCP)] [answered]
According to [this](https://stackoverflow.com/questions/2672734/tcp-sequence-number-question) post, the next sequence number after 4294967295 (which is 2^32 - 1) is 0.

Do I need to consider sequence number overflow in PA3, or can I just assume that there are no overflows in the test cases?

**Comment by Pusnow** [comment, marked as answer]
Yes, you should consider the overflow. We would not explicitly test overflows, but they would happen depending on RANDOM_SEED.
